news/2017-09-21-osu-mania-4k-world-cup-results.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "osu!mania 4K World Cup - Results"
+date: 2017-09-21 00:01:00 +0000
+---
+
+At long last, we've finally reached the end of the osu!mania 4K World Cup! Here's to another year of keysmashing, as we dive right into the thrilling conclusion!
+
+![](/wiki/shared/news/banners/mania4k_logo_2017.jpg)
+
+### Navigation
+
+<a href="#results">Results</a></br><a href="#recap">Recap</a>
+* <a href="#mappool">Mappool</a>
+* <a href="#matches">Tournament Matches</a>
+
+--------
+
+Over the course of osu!mania 4K World Cup history, never has there been a tournament more spectacular or successful. Consisting of 78 intense matches over 6 gruelling weeks, the top 32 teams from all around the world did battle, and were undoubtedly pushed to their limits. However, only one team would stand triumphant in the face of these difficult trials. 
+
+Let's not wait any longer - it's time to announce the results of the osu!mania 4K World Cup!
+
+## <a name="results" id="results"></a>Results
+
+![](/wiki/shared/news/2017-09-21-osu-mania-4k-world-cup-results/podium.jpg)
+
+**With a nearly flawless winning streak, team South Korea are crowned the osu!mania 4K World Cup 2017 Champions!**
+
+Team captain [jakads](https://osu.ppy.sh/users/259972) returns again to lead South Korea towards victory for their second year in a row! Despite being only a 5-man team, their impressive skill and teamwork was what allowed them to prevail. Amazingly, they did not concede a single point at all during the elimination stages. They will proudly carry the title of osu!mania champions for another year!
+
+South Korea takes home the grand prize - $150 per team member, exclusive osu! pins and merchandise, a unique profile badge, and the highly coveted "osu!mania Champion" user title for one year! Many congratulations to them!
```

Comment:
add mention that everyone who supported SK with a profile banner now gets the HIGHLY COVETED WINNER'S BANNER until the next tournament starts